DEPS

```diff
@@ -17,7 +17,7 @@ deps = {
   },
   "vendor/bat-native-tweetnacl": "https://github.com/brave-intl/bat-native-tweetnacl.git@800f9d40b7409239ff192e0be634764e747c7a75",
   "vendor/gn-project-generators": "https://github.com/brave/gn-project-generators.git@b76e14b162aa0ce40f11920ec94bfc12da29e5d0",
-  "vendor/web-discovery-project": "https://github.com/brave/web-discovery-project@3aa5b8bcc8494aa2aca5fc10938443886c36b165",
+  "vendor/web-discovery-project": "https://github.com/brave/web-discovery-project@7159cb52cf674da3aed53ad0ab757513703a06eb",
```

Comment:
can you link in the PR description to the relevant PR for web-discovery-project?

Reply:
done